safe/report/expressions/map_report.py

```diff
@@ -208,7 +278,7 @@ def bearing_to_nearest_place(feature, parent):
 @qgsfunction(
     args='auto', group=label_group, usesGeometry=False, referencedColumns=[],
     help_text=help_message.to_html(), helpText=help_message.to_html())
-def name_of_the_nearest_place(feature, parent):
+def name_of_the_nearest_place(exposure_key, feature, parent):
```

Comment:
So we still have the exposure_key? Even if you added the logic with the selectedlayer?

Reply:
my fault